src/lib/Account.js

```diff
@@ -125,6 +125,9 @@ export default class Account {
 
   async sync () {
     try {
+      if ('syncStart' in this.server)
+        await this.server.syncStart ();
+
       if (this.getData().syncing) return
       console.log('Starting sync process for account ' + this.getLabel())
       await this.setData({...this.getData(), syncing: true})
```

Comment:
`syncStart` should be triggered after this block, to make sure that no two tasks are syncing the same account ;)